### ISSUE TITLE
Importing evernote notes from .enx extension does not import anything

### ISSUE THREAD

**Issue by sammcj** [HappenApps/Quiver#416]
1. I export my notes from evernote as an .enex file.
2. I select import -> from .enex file and select the file.
3. Quiver pauses for a second, then creates a notebook with the import name.
4. The notebook is empty and contains no notes.
- The enex file contains 1157 notes.
- The enex file is is 400MB in size.

As there is no way to import HTML notes it means I cannot migrate from Evernote.

**Comment by ylian** [comment]
Try the build in https://github.com/HappenApps/Quiver/issues/361

**Comment by sammcj** [comment]
@ylian thanks but no difference.

![screen shot 2016-02-05 at 12 29 57 pm](https://cloud.githubusercontent.com/assets/862951/12835435/37087c28-cc04-11e5-9dd7-b7b866d2d6d7.jpg)

**Comment by ylian** [comment]
You have to try to import again using the new build. If it still doesn't work, maybe send me the ENEX file (or a portion of it) at ylian@happenapps.com. I will figure out what's stopping the importer.

**Comment by sammcj** [comment]
Yeah I did that, it looks like if I export just a few at a time it seems to import alright.

My ENEX is almost 500MB so I wonder if something is borking there.

What I'll do so export my Evernote notes in chunks, and import them bit by bit - when (if) I come to one that won't import - I'll export that and give it to you for analysis.

**Comment by ylian** [comment]
That would be fantastic. There is obviously a problematic note.

**Comment by sammcj** [comment]
OK you're really not going to like this bug...

I imported my 1106 notes notebook by notebook.
Some notebooks imported fine in one go, they contained anywhere from 1 to 115 notes.

Some failed and I had to break them up into logs of between 25 to 150 at a time, I could not find any correlation between the number of notes imported so I have a feeling its when you import a certain two notes at once and they clash some home.

I'm happy now has all my notes imported fine, hopefully it doesn't happen for anyone else.
